packages/p2p/src/components/orders/order-table/order-table.jsx

```diff
@@ -7,22 +7,17 @@ import OrderTableContent from './order-table-content.jsx';
 import { orderToggleIndex } from '../order-info';
 
 const OrderTable = ({ showDetails }) => {
-    const {
-        active_notification_count,
-        inactive_notification_count,
-        order_table_type,
-        changeOrderToggle,
-    } = React.useContext(Dp2pContext);
-    const orders_filters = [
+    const { LocalStorage, changeOrderToggle, order_table_type } = React.useContext(Dp2pContext);
+    const getOrderFilters = () => [
         {
             text: localize('Active order'),
             value: orderToggleIndex.ACTIVE,
-            count: active_notification_count,
+            count: LocalStorage?.getActiveNotificationCount?.(true) || 0,
```

Comment:
`getActiveNotificationCount` wont be undefined since it is a class method. Can we remove optional chaining for it?

Reply:
`LocalStorage` may be undefined as it isn't required to be passed by the P2P consumer.

Reply:
I'm referring to optional chaining in `getActiveNotificationCount?.(true)` function call